kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/README.md

```diff
@@ -5,9 +5,9 @@ The sample scripts demonstrate the creation of a WebLogic domain home in a Docke
 ## Prerequisites
 
 The following prerequisites must be handled prior to running the create domain script:
-* The WDT sample requires that JAVA_HOME is set to a java JDK version 1.8 or greater
-* Make sure the WebLogic operator is running.
-* The operator requires WebLogic Server 12.2.1.3.0 with patch 28076014 applied. Refer to [Weblogic Docker images](../../../../../site/weblogic-docker-images.md) for details on how to create one. If a different `domainHomeImageBase` (see Configuration table below) is specified, the specified image needs to be built locally or pulled from a repository.
+* The WDT sample requires that `JAVA_HOME` is set to a Java JDK version 1.8 or later.
+* Make sure that the WebLogic operator is running.
+* The operator requires WebLogic Server 12.2.1.3.0 with patch 28076014 applied. Refer to [WebLogic Docker images](../../../../../site/weblogic-docker-images.md) for details on how to create one. If a different `domainHomeImageBase` (see Configuration table below) is specified, the specified image needs to be built locally or pulled from a repository.
 * Create a Kubernetes namespace for the domain unless the intention is to use the default namespace.
 * Create the Kubernetes secrets `username` and `password` of the admin account in the same Kubernetes namespace as the domain.
```

Comment:
Not directly related to the current changes, but the 'secrets' pre-req looks a bit confusing, as it implies you need to create secrets actually named 'username' and 'password'.   How about something like:

Create a Kuberenetes secret for the WebLogic administrator credentials that contains fields `username` and `password`, and make sure the secret name matches the value specified for `weblogicCredentialsSecretName` (see Configuration table below).  For example:

   1) cd ./kubernetes/samples/scripts/create-weblogic-domain-credentials
   2) create-weblogic-credentials.sh \
      -u weblogic -p welcome1 -d domain1 -n default \
      -s domain1-weblogic-credentials
   3) make sure `weblogicCredentialsSecretName` is configured to be `domain1-weblogic-credentials`

Reply:
Everything else LGTM.